src/main/java/com/spotify/docker/client/messages/Info.java

```diff
@@ -449,7 +449,7 @@ public String toString() {
 
   public static class IndexConfig {
     @JsonProperty("Name") private String name;
-    @JsonProperty("Mirrors") private String mirrors;
+    @JsonProperty("Mirrors") private List<String> mirrors;
```

Comment:
Seems like it can't hurt? I can fix this in a later commit. I noticed we aren't consistent about this.